lib/serializer/BinaryDeserializer.cpp

```diff
@@ -98,7 +97,7 @@ void CLoadFile::clear()
 
 void CLoadFile::checkMagicBytes(const std::string &text)
 {
-	std::string loaded = text;
+	const std::string& loaded = text;
```

Comment:
Change of behavior - now `read()` will overwrite `text` since they both point to same variable.
Is this clang-tidy change or manual one?

Reply:
clang-tidy one.

Reply:
reverted